<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Script</class>
 <widget class="QDialog" name="Script">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>宏</string>
  </property>
  <widget class="QPushButton" name="pushButtonCopy_1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>141</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextEdit" name="textEditInstruct1"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditInstruct2"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditInstruct3"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonCopy_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>211</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEditScript1"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditScript2"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditScript3"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonCopy_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
